support/Pipelines/Bbh/ControlId.py

```diff
@@ -18,9 +18,37 @@
 DEFAULT_RESIDUAL_TOLERANCE = 1.0e-6
 DEFAULT_MAX_ITERATIONS = 30
 
+# Initial data physical parameters that are supported in this control scheme
+SupportedParams = Literal[
```

Comment:
(optional) You can do this to avoid duplicating this list:

```py
SupportedParams = Literal[*FreeDataFromParams.keys()]
```

Reply:
It seems like `Literal` doesn't take "unpacked arguments", so this would disable the type checking. Since these parameters are specified by users in the input file, I think it would be better to keep it the way it is.

Reply:
Oh ok it worked on my machine with Py 3.12, but maybe this was a more recent addition to Python. Fine with me.